src/DeviceInput/index.ts

```diff
@@ -1 +1,3 @@
 export * from "./deviceInputSystem";

```

Comment:
Do we need have some imports here before the exports to ensure these new types don't get stripped out? cc @bghgary

Reply:
Probably. If types don't get used, they won't make it into the js without an additional import.

Reply:
Added import